src/components/table/table.scss

```diff
@@ -69,6 +69,11 @@ $tabulator-arrow-color-active: var(--lime-turquoise);
         background-color: #fff;
     }
 
+    &.tabulator-selected,
+    &.tabulator-selected:hover {
+        background-color: rgba(#26a69a, 0.16) !important;
```

Comment:
It would be great if we can do this without `!important` :slightly_smiling_face:

Reply:
First: change the class name from `.tabulator-selected` to `.activated`. I think it's good to remove the word `tabulator` from the class name, so we know in future that this is not a native Tabulator function or something.

Reply:
The `!important" is used because there is already a `background-color` style for normal `:hover` on line 60.

So, either we should make this rule more specific here. For example: 

````css
#tabulator-table {
    .tabulator-table {
        .tabulator-row {
            &.tabulator-selected,
            &.tabulator-selected:hover {
                 background-color: rgba(#26a69a, 0.16);
            }
        [...]
````
or add another identifier for tables that have this capability ON, e.g. "`.has-activateable-rows`" and make the CSS bound to that.

**OR this is preferred solution according to me:**
refine the code for hover on line 57, like this:
````css
.tabulator-row {
    &.tabulator-selectable {
        transition: background-color 0.2s ease;
        &:not(.activated) {
            &:hover {
                background-color: $table-header-background-color;
            }
        }
    }
````
Note: Test the above code. I guess I've written it correctly, but make sure to test.